include/InterfaceParams.h

```diff
@@ -1,14 +1,17 @@
 #ifndef INTERFACEPARAMS
 #define INTERFACEPARAMS
-#include "NativeEthernet.h"
+
+#include <QNEthernet.h>
+
+using namespace qindesign::network;
 
 namespace EthParams
 {
     uint8_t default_MCU_MAC_address[6] = 
     {0x04, 0xe9, 0xe5, 0x10, 0x1f, 0x22};
 
     const IPAddress default_MCU_ip(192, 168, 1, 30);
-    const IPAddress default_TCU_ip(192, 168, 1, 68);
+    const IPAddress default_TCU_ip(192, 168, 1, 69);
```

Comment:
is there a bookstack page with the IP addresses of all ecus? If not, would be nice.

Reply:
here it is:

https://wiki.hytechracing.org/books/software/page/ip-addresses-and-ports-of-software-and-firmware